### ISSUE TITLE
About Hybrid Enhanced Robust Optimization

### ISSUE THREAD

**Issue by Just-ZJJ** [hero-slam/HERO-SLAM#2]
Dear Author,

Hello, regarding the "About Hybrid Enhanced Robust Optimization" section, I have a question for you. 
In the file heroslam_opt_pose_adaptive_interval.py at line 848, I noticed the following code:
`
 if self.config['loss']['use_warp_loss']:
                # # warping loss + rendering loss
                loss = 0.0
                loss_pose, loss_kp_mean = self.get_loss_from_match(mkpts_cur, mkpts_last, c2w_gt, batch, mscores, i)
`
`
self.get_loss_from_match(mkpts_cur, mkpts_last, c2w_gt, batch, mscores, i)
`

You used c2w_gt as the pose for the current frame. Why not use c2w_est instead?
